dockernet/build.sh

```diff
@@ -21,7 +21,7 @@ build_local_and_docker() {
    cd $folder
 
    # Clear any previously build binaries, otherwise the binary can get corrupted
-   if [[ "$module" == "stride" ]]; then
+   if [[ "$module" == "comdex" ]]; then
```

Comment:
@Surendra-Patil this needs to be reverted, it will break dockernet testing. This portion of build.sh is concerned with strided, not the host zone binaries